src/isPalindrome.js

```diff
@@ -19,7 +19,12 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  if (str === '') {
+    return true;
+  }
+  const palindrome = str.toUpperCase().match(/\w/ig).reverse('').join('');
+  const newStr = str.toUpperCase().match(/\w/ig).join('');
```

Comment:
you can use `newStr`. It's better for performance

```suggestion
  const newStr = str.toUpperCase().match(/\w/ig);
  const palindrome = newStr.reverse('').join('');
```

Reply:
it is really better

Reply:
yes, it's my fall.
Now, `newStr` is array, you need to convert it to the string.